src/lib/ci.js

```diff
@@ -9,15 +9,17 @@ const { getChangedFiles } = require("./utils");
  */
 async function isCIRequired(context, diff_url) {
   const triggerPaths = await context.config("bot-files/paths.yml");
-  const changedFiles = await getChangedFiles(diff_url);
+  if (triggerPaths !== null) {
```

Comment:
I would replace `if (triggerPaths !== null)` by `if (!triggerPaths)`

Reply:
Will be `if(triggerPaths)`